src/parameterized/gamma.jl

```diff
@@ -0,0 +1,27 @@
+# Gamma distribution
+
```

Comment:
I was just thinking it would be helpful to add some references to the tops of the files, something like
```suggestion
# REFERENCES
# https://mc-stan.org/docs/2_27/functions-reference/gamma-distribution.html
# https://en.wikipedia.org/wiki/Gamma_distribution
# https://juliastats.org/Distributions.jl/stable/univariate/#Distributions.Gamma
```